docs/proposals/capacity-planning-refactoring.md

```diff
@@ -0,0 +1,69 @@
+# Proposal: Refactoring of capacity planning
+
+Last updated: 09.08.2021
+
+
+## Abstract
+
+Refactor logic of AC reservation to consider cases for 2 or more LVG PVs
+
+## Background
+
+Currently, Reservation Controller doesn't consider LVG volumes on the planning phase. 
```

Comment:
This statement is incorrect. It does consider but due to AC conversion to LVG we can use more than 1 disk.

Reply:
updated